run.py

```diff
@@ -4,14 +4,15 @@
 from controllers import db
 
 
-from models.prediction_model import Prediction
-from models.qa_model import QAModel
-# from models.visitor_model import Visitor
+# from models.prediction_model import Prediction
+# from models.qa_model import QAModel
+# # from models.visitor_model import Visitor
 
```

Comment:
If the commented out lines don't need to be a part of the code, they should just be removed entirely (as in, just delete those lines).   Keeping all of those commented out lines adds confusion.  When we go back later we will have no idea if they can stay or go.